packages/wonder-blocks-breadcrumbs/components/breadcrumbs.test.js

```diff
@@ -0,0 +1,27 @@
+// @flow
+import * as React from "react";
+import {mount, unmountAll} from "../../../utils/testing/mount.js";
+import Breadcrumbs from "./breadcrumbs.js";
+import BreadcrumbsItem from "./breadcrumbs-item.js";
+
+describe("Breadcrumbs", () => {
+    afterEach(() => {
+        unmountAll();
+    });
+
+    it("should set aria-current to the last item", () => {
+        // Arrange
+        const wrapper = mount(
+            <Breadcrumbs>
```

Comment:
Can you add a test to verify `data-test-id` gets set when `testId` is present on `Breadcrumbs`?

Reply:
Added a second unit test on line # 28

Reply:
Thanks!